assets/js/amber.js

```diff
@@ -226,3 +226,18 @@ export class Socket {
 module.exports = {
   Socket: Socket
 }
+
+
+/**
+ * Allows delete links to post for security and ease of use similar to Rails jquery_ujs
+ */
+$(document).ready(() => {
+  $( "a[data-method='delete']" ).click(function(){
```

Comment:
This would mean that any element in the page will behave like a delete button. What if the intention is not use it as such. This is making a lot of assumptions and is very particular to someone specific use case.

Reply:
Only elements with `data-method=delete`. Every other element will NOT behave like a delete button.

Reply:
Hey @elorest What about adding a `data.message` attribute ?

so We would be able to do this:

```html
<a href="/post/delete/1"
    data-method="delete"
    data-message="¿Estás seguro que deseas borrar esto?">Borrar</a>
```

WDYT?